src/postprocessors/NekInfoPostprocessor.C

```diff
@@ -0,0 +1,300 @@
+    if (platform->options.compareArgs("VELOCITY BLOCK SOLVER", "TRUE"))
+      return nrs->uvwSolver->Niter;
+    else if (nrs->uSolver)
+      return nrs->uSolver->Niter;
+    else
+      return -1;
+  }
+  
+  // Pressure iterations
+  else if (_test_type == "n_iter_pressure")
+    return nrs->pSolver ? nrs->pSolver->Niter : -1;
+  
+  // Temperature iterations
+  else if (_test_type == "n_iter_temperature") {
+    if (platform->options.compareArgs("SCALAR00 SOLVER", "CVODE")) {
+      return -1;
+    }
+    else if (platform->options.compareArgs("SCALAR00 SOLVER", "NONE"))
+      return -1;
+    else
+      return nrs->cds->solver[0]->Niter;
+  }
+  
+  // Scalar 01 iterations
+  else if (_test_type == "n_iter_scalar01") {
+    if (platform->options.compareArgs("SCALAR01 SOLVER", "CVODE")) {
+      return -1;
+    }
+    else if (platform->options.compareArgs("SCALAR01 SOLVER", "NONE"))
+      return -1;
+    else
+      return nrs->cds->solver[1]->Niter;
+  }
+  
+  // Scalar 02 iterations
+  else if (_test_type == "n_iter_scalar02") {
+    if (platform->options.compareArgs("SCALAR02 SOLVER", "CVODE")) {
+      return -1;
+    }
+    else if (platform->options.compareArgs("SCALAR02 SOLVER", "NONE"))
+      return -1;
+    else
+      return nrs->cds->solver[2]->Niter;
+  }
+  
+  // Scalar 03 iterations
+  else if (_test_type == "n_iter_scalar03") {
+    if (platform->options.compareArgs("SCALAR03 SOLVER", "CVODE")) {
+      return -1;
+    }
+    else if (platform->options.compareArgs("SCALAR03 SOLVER", "NONE"))
+      return -1;
+    else
+      return nrs->cds->solver[3]->Niter;
+  }
+  
+  // Time step
+  else if (_test_type == "tstep")
+    return nrs->tstep;
+  
+  // CFL number
+  else if (_test_type == "cfl")
+    return computeCFL(nrs);
+  
+  // Termodynamic pressure
+  else if (_test_type == "p0th")
+    return nrs->p0th[0];
+  
+  // Termodynamic pressure time derivative
+  else if (_test_type == "dp0thdt")
+    return nrs->dp0thdt;
+  
+  // Velocity solver
+  else if (_test_type == "solver_velocity"){
+    if (platform->options.compareArgs("VELOCITY SOLVER", "NONE"))
+      return 0;
+    else
+      return 1;
+  }
+  
+  // Temperature solver
+  else if (_test_type == "solver_temperature"){
+    if (platform->options.compareArgs("SCALAR00 SOLVER", "NONE"))
+      return 0;
+    else
+      return 1;
+  }
+  
+  // Scalar01 solver
+  else if (_test_type == "solver_scalar01"){
+    if (platform->options.compareArgs("SCALAR01 SOLVER", "NONE"))
+      return 0;
+    else
+      return 1;
+  }
+  
+  // Scalar02 solver
+  else if (_test_type == "solver_scalar02"){
+    if (platform->options.compareArgs("SCALAR02 SOLVER", "NONE"))
+      return 0;
+    else
+      return 1;
+  }
+  
+  // Scalar03 solver
+  else if (_test_type == "solver_scalar03"){
+    if (platform->options.compareArgs("SCALAR03 SOLVER", "NONE"))
+      return 0;
+    else
+      return 1;
+  }
+  
+  // Mesh - X minimum value
+  else if (_test_type == "xmin") {
```

Comment:
FYI these mesh operations will not work if running in parallel - I think these all need MPI communication to find the min/max coordinate across all ranks. Will fix this in the new version.